docs/select-pool-to-delegate.md

```diff
@@ -1,53 +1,75 @@
+Delegation allows holders of ada to transfer their right to participate in the proof of stake protocol to stake pools. Stake pools are run by stake pool operators (SPOs) and a person delegating to a stake pool is called delegator, member, or participant. A stake pool consists of a block producer (BP) and any number of relays - each of these are called a node. The blockchain is made up of consecutive blocks and stake is power to make blocks.
 
-There is no 1-for-all pool that you can delegate with, otherwise - every one would be delegating to same pool and there wouldnt be a point of decentralisation :) . While many users are attracted to the pool with lowest fee margins, it is usually the least important factor to distinguish between stake pools. Below are a few factors that might be important to take into consideration.
+If one pool or person is given too much stake power they can control consensus, engage in double-spending attacks, create forks, censor blocks, and damage or even destroy the system. Your delegation choice, who you give your power to, is very important at limiting a single entity from creating too many blocks.
 
-!!! danger ""
+“There should never be a conflict of interest between maximizing rewards and ‘doing the right thing’”
 
-    Remember, you - a delegator - are indirectly one of the core components of the protocol and your actions may determine the decentralisation as well as adoption of the platform. Whether it is for your benefit, or benefit of the network - it is important that you select carefully who you will be delegating to.
+The staking mechanism was designed so that you can optimally secure the network by trying to maximize your rewards and being greedy with your delegation choice. Stake pools can offer very different return rates.
 
-!!! tip ""
+“How attractive a pool is to delegators depends on four interacting elements: pledge (the higher, the better); operating costs (the lower, the better); operator margin (the lower, the better); performance (the higher, the better).”
 
-    There are heavy contributors to the ecosystem that may not be advertising and do not incur enough delegations, you can read more about them [here](community-contrib.md) if you'd like to support them.
+It is important that you understand each of these elements and how they can effect your rewards.
 
-In this section we will discuss the different aspects that users should consider when selecting who they would like to support. If you're unsure about any of the parameters when simply using wallet, please leverage benefits of community platforms built to help you with the decision making. The platforms are:
+The first thing you should consider when looking for a pool is pledge. Pledge is the amount of stake the SPO has in their own pool.
 
-  * [Pooltool](https://pooltool.io)
```

Comment:
Dont think the explorer references should be removed, an end user wouldn't know where to look at and compare between pools